src/metrics/winogender_metrics.py

```diff
@@ -0,0 +1,44 @@
+class GenderParity:
+    """
+  Gender parity metric from [Gender Bias in Coreference Resolution](https://arxiv.org/pdf/1804.09301.pdf).
```

Comment:
I can't find any mention of this in the paper—it might just be on the site. If so, link to the site instead.